scripts/exgdas_global_marine_analysis_prep.py

```diff
@@ -161,7 +161,8 @@ def gen_bkg_list(bkg_path, out_path, window_begin=' ', file_type='gdas.t*.ocnf00
 ufsda.mkdir(bkg_dir)
 
 # create output directory for soca DA
-anl_out = os.path.join(comout, 'ocnanal_'+os.getenv('CDATE'), 'Data')
+cdate = os.getenv('PDY')+os.getenv('cyc')
+anl_out = os.path.join(comout, 'ocnanal_' + cdate, 'Data')
```

Comment:
I know this is @guillaumevernieres 's problem, not yours, but `$COMOUT` should not have `ocnanal_`... in it. Just making a note here.

Reply:
Not sure I understand your comment @CoryMartin-NOAA `$COMOUT` doesn't have `ocnanal_...` in it ... Or do you mean anl_out should be something else?

_I'm being generous and decided to share some of my problems with @AndrewEichmann-NOAA_

Reply:
if you look at like the existing atmos GDAS ROTDIR, there is no subdirectory for analysis stuff, it is all there together. Not sure this is a 'requirement' or not, just thinking of consistency.

Reply:
OK, save as an issue @CoryMartin-NOAA .